.github/workflows/landing-builds.yml

```diff
@@ -14,7 +14,9 @@ on:
       - 'utils/scripts/helpers/*'
       - utils/ci
       - ci
-      - requirements.txt
+      - requirements-build
```

Comment:
```suggestion
      - requirements-build.txt
```
?

Reply:
Hrm.  Perhaps not.  But `- requirements-build` doesn't seem right when before it was `- requirements.txt`.

Reply:
No, you're right.  This should have a txt suffix.